### DISCUSSION TITLE
Not being able to acquire data locally

### DISCUSSION THREAD

**Discussion by coloal** [dcaribou/transfermarkt-datasets#86] [Q&A] [answered]
1_acquire.py file is not getting any data. I have run:
python 1_acquire.py local --asset clubs
python 1_acquire.py local --asset all --season 2021

And none of them starts producing any json. It only outputs a message: 
"Schedule clubs" or "Schedule games"
But after some minutes of running it does not advance from there and if you check "./data/raw/" nothing has been created. I am running it from a CMD.exe prompt opened via Anaconda. Any help on this? It seems like some infinite loop. To get to this point I had to previously create the directory "./data/raw/2021", if not the scripts complained about not finding the route.

**Comment by dcaribou** [comment]
Hi @coloal.
Thanks for raising this.
The script does take some time to scrape the data, but after some minutes you should indeed see something in data/raw/2021.
Can you try activating the debug log for scrapy, so we can see more logs? You should be able to do this by changing the log level to DEBUG [here](https://github.com/dcaribou/transfermarkt-scraper/blob/ba1f2e61ae26dd9e492433b51d955b7774919d4d/tfmkt/settings.py#L28)

**Comment by coloal** [comment, marked as answer]
Ok, super silly problem. I cloned this project but I did not clone the transfermarkt-scraper, so I had an empty directory instead. I suspect is a git thing I am unaware of because of my inexperience but do you know how would I have got this dependency in the first place, not having to do it manually?

**Reply by dcaribou** [reply]
Ah yes, you have to clone it with the --recursive option, so the submodule gets pulled as well.
So `git clone --recursive <repo url>`

**Reply by coloal** [reply]
Ok, thanks a lot!

**Reply by coloal** [reply]
Just tested the `git clone --recursive <repo url>` flag and I get:

Submodule 'transfermarkt-scraper' (git@github.com:dcaribou/transfermarkt-scraper.git) registered for path 'transfermarkt-scraper'
Cloning into 'C:/Users/alber/FootDataAnalysis/transfermarkt-datasets/transfermarkt-scraper'...
Host key verification failed.
fatal: Could not read from remote repository.

**Reply by dcaribou** [reply]
Have you tried with
`clone --recursive git@github.com:dcaribou/transfermarkt-datasets.git`
?
This works for me with git version 2.36.0

Maybe [this](https://stackoverflow.com/questions/13363553/git-error-host-key-verification-failed-when-connecting-to-remote-repository) is helpful too.
